docs/data/material/components/autocomplete/autocomplete.md

```diff
@@ -227,6 +227,11 @@ The `renderInput` prop allows you to customize the rendered input.
 The first argument of this render prop contains props that you need to forward.
 Pay specific attention to the `ref` and `inputProps` keys.
 
+:::warning
+If you're using a custom input component inside the Autocomplete, you might need to include `forwardRef`.
+When incorporating the [Text Field](/material-ui/react-text-field/) component, you should pass `ref` to `inputRef`.
```

Comment:
```suggestion
When using the [Text Field](/material-ui/react-text-field/) component, you should pass `ref` to `inputRef`.
```

Reply:
This is a note for a pit fall. You only need to pass spreaded `params` into TextField if it is mui's, but other are not